jbpm-console-ng-process-runtime/jbpm-console-ng-process-runtime-client/src/main/java/org/jbpm/console/ng/pr/client/editors/instance/details/ProcessInstanceDetailsWholeViewImpl.java

```diff
@@ -0,0 +1,71 @@
+package org.jbpm.console.ng.pr.client.editors.instance.details;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+
+import org.jbpm.console.ng.pr.client.i18n.Constants;
+
+import com.github.gwtbootstrap.client.ui.ControlLabel;
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.uibinder.client.UiBinder;
+import com.google.gwt.uibinder.client.UiField;
+import com.google.gwt.user.client.ui.Composite;
+import com.google.gwt.user.client.ui.HTMLPanel;
+import com.google.gwt.user.client.ui.Widget;
+
+@Dependent
+public class ProcessInstanceDetailsWholeViewImpl extends Composite implements ProcessInstanceDetailsPresenter.ProcessInstanceDetailsWholeViewUpdate {
+
```

Comment:
Why is this class called WholeView? We should have an Advanced and Basic version of the screen right?